### DISCUSSION TITLE
autostart Frigate after OS boot

### DISCUSSION THREAD

**Discussion by sudo-joe** [blakeblackshear/frigate#4168] [General]
Hello,

e.g. in the case of a power outage or other things, I am wondering if there is a way to have Frigate auto-started after Linux has booted... (without beeing required to login)?

It would be nice, if this behaviour could mostly be configured in Frigate and not in Linux:)

Thank's

**Comment by NickM-27** [comment]
Frigate runs in docker so it is configured in docker. Simply set the restart mode to `always` or `unless-stopped`

**Reply by NickM-27** [reply]
https://docs.docker.com/config/containers/start-containers-automatically/
